doc/rds.md

```diff
@@ -0,0 +1,31 @@
+# Amazon RDS
+
+## Limitations
+
+- No `SUPER` privileges.
+- `gh-ost` runs should be setup use [`--assume-rbr`][assume_rbr_docs] and use `binlog_format=ROW`.
```

Comment:
I gather you mean, "whoa, where's the actual link? have you gone crazy?" 😛 

These are [reference links](https://daringfireball.net/projects/markdown/syntax#link) which allow you to reuse a single link multiple times without repeating it over and over. The links themselves are put at the [bottom](https://github.com/github/gh-ost/pull/385/files#diff-82208f0a804594829ab45d4f59e7d19fR28).

Or would you rather not use reference links?